statistics-dashboard-common/src/main/java/eu/europeana/statistics/dashboard/common/api/request/StatisticsRangeFilter.java

```diff
@@ -1,12 +1,15 @@
 package eu.europeana.statistics.dashboard.common.api.request;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.apache.commons.lang3.StringUtils;
 
 /**
  * Class that represents a filter type range.
  * That is, a filtering that contains range conditions
  */
 
+@JsonIgnoreProperties(value = { "valuesEmpty" })
```

Comment:
In both `StatisticsRangeFilter` and `StatisticsValueFilter` there is a boolean method that come from the interface `StatisticsFilter` called `isValuesEmpty`. Somehow, when using these classes for the response of available filtering options, a new field called `valuesEmpty` appears, and we don't want that, so I added this annotation so the field `valuesEmpty` does not show in the Json object either in request or response

Reply:
Ahh I see, I misinterpreted this then.
Okay in this case perhaps it's better to place the `JsonIgnore` to the method, to avoid typing the constant string value on the current setup?

Reply:
Good point, I'll do that then